modules/ocf_mirrors/manifests/projects/alpine.pp

```diff
@@ -0,0 +1,25 @@
+class ocf_mirrors::projects::alpine {
+  file {
+    '/opt/mirrors/project/alpine':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/alpine',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  ocf_mirrors::monitoring { 'alpine':
+    type          => 'unix_timestamp',
+    upstream_host => 'mirrors.alpinelinux.org',
+    ts_path       => 'TIME',
+  }
+
+  ocf_mirrors::timer {
+    'alpine':
+      exec_start => '/opt/mirrors/project/alpine/sync-archive',
+      hour       => '0/3',
+      minute     => '22',
```

Comment:
This will run every 3 hours, but they seem to want us to sync every hour, which is probably fine. We can use `'*'` instead of `'0/3'`for `hour`. Also, centos already syncs at minute 22, so maybe we can choose another value random value for that.

Reply:
Ah Mb, didn't know minute should be random :( In that case we should probably change here too https://github.com/ocf/puppet/blob/4e37cbedd228d89bc2f32234dbb4fc54114faa9d/modules/ocf_mirrors/manifests/projects/archlinux.pp#L21